mobile/src/components/common/search/search.tsx

```diff
@@ -1,8 +1,6 @@
 import React, { FC } from 'react';
 import { ImageURISource, TextInput, TouchableOpacity } from 'react-native';
 
-import search from '~/assets/images/search.png';
-import voice from '~/assets/images/voice.png';
```

Comment:
You can just wait for merging my PR (sign-up), there I have a file, that allows you make this imports

Reply:
As far as I understand this is a temporary solution.
When the icon component is ready, these imports will be removed in favor of using the icon component